doc/nrf/libraries/others/flash_map_pm.rst

```diff
@@ -0,0 +1,55 @@
+.. _lib_flash_map_pm:
+
+Partition Manager flash map
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Partition Manager flash map library provides a set of defines for reading flash partition information when using the :ref:`partition_manager` script.
+This is mandatory for providing a unified interface for flash partitions when Partition Manager is used.
+
+Overview
+********
+
+In Zephyr, :file:`flash_map.h` provides unified interface to code for reading the flash area data, such as offset, size, and device.
+By default, the header file gets this flash area information from :ref:`zephyr:dt-guide` (namely, :file:`zephyr.dts`).
+
+In the |NCS|, when Partition Manager is used, flash area information is not taken from Devicetree.
```

Comment:
If it's company style to treat 'devicetree' as a proper noun, then please ignore me. But the usual style is to just say 'devicetree', not 'Devicetree'.